src/crystal/system.cr

```diff
@@ -7,10 +7,15 @@ module Crystal
     # Returns the number of logical processors available to the system.
     #
     # def self.cpu_count
+
+    # Returns the short user name of the currently logged in user.
+    #
```

Comment:
I think that the comment should be adjacent to the `def self.login` as with `hostname`. Please can you also modify `cpu_count` at the same time, as this is a tiny change not worth a new PR.